_posts/2021-10-14-Subnero-at-Ocean-Business.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Subnero attended Ocean Business 2021 in Southampton, UK
```

Comment:
I would suggest to remove the place and country from the title as it would be a bit long and have it same as other similar posts. Maybe can change it to something like 'Subnero at Ocean Business 2021'.

Reply:
Agree, I'll amend accordingly.